src/lib/classes/saml_settings.php

```diff
@@ -231,11 +292,34 @@ private function _get_settings()
     else
     {
       $this->settings = $this->_use_defaults();
+      // In multisite, copy the idp details from main blog
+      if (is_multisite()) {
+          $this->_copy_main_idp_details();
```

Comment:
@ktbartholomew I added this hack to copy the idp details from the main blog. This will get out of sync. I was wondering is there a reason why users are not allowed to edit the idp details? If we want to reference the main blog idp details directly, I will need to update a few places to use this same multisite switch.